RELEASE-NOTES.txt

```diff
@@ -2,7 +2,7 @@
 
 16.9
 -----
-
+- [*] Order creation: after selecting a registered customer, the customer is now linked to the order. [https://github.com/woocommerce/woocommerce-ios/pull/11645]
```

Comment:
Do we need to add all pull requests info to release notes or just new features? I might have missed this in the docs

Reply:
Just user-facing or internal changes that might be worth noting for regression testing.

This might not be mentioned in a doc, there's a checkbox in the PR template as a reminder - `I have considered if this change warrants user-facing release notes and have added them to RELEASE-NOTES.txt if necessary.`. The release notes are useful for:
- The editorial team to come up with the public release notes, announcing any user-facing features
- The quality & engineering team to test user-facing and risky changes like from a big refactoring (usually noted with [Internal]). We no longer have a QA process, but some teams like Kiwi test changes made by the team in every release and this is how we keep track of note-worthy changes

Reply:
> there's a checkbox in the PR template as a reminder

Oh yes, totally forgot about that one, tnx! :)